include/openmc/mgxs_interface.h

```diff
@@ -12,36 +12,71 @@
 namespace openmc {
 
 //==============================================================================
-// Global variables
+// Global MGXS data container structure
 //==============================================================================
 
-namespace data {
+struct MgxsInterface
```

Comment:
As @smharper discussed, usually `struct` implies the class is relatively simple. I would make this `class`

Reply:
Makes sense to me; I haven't thought about it that way.